CHANGELOG.md

```diff
@@ -106,6 +106,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
 - Fixed an issue with `training_step` outputs not getting collected correctly for `training_epoch_end` ([#8613](https://github.com/PyTorchLightning/pytorch-lightning/pull/8613))
+- Fixed avoid wrapping LightningModule in *DataParallel overrides when not fitting ([#6977]https://github.com/PyTorchLightning/pytorch-lightning/issues/6977)
```

Comment:
- Please make the changelog explicit with which plugins and data-parallel wrappers were changed to help users who search this
- Please add the changelog entry at the end of the list, along with the same formatting that other entries have

Reply:
I updated changelog with more details in the coming PR.